smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/ConfigField.java

```diff
@@ -41,7 +73,11 @@ public enum Type {
         /**
          * Specifies the property is auxiliary, e.g. {@code region} for {@code @aws.auth#sigv4}
          */
-        AUXILIARY
+        AUXILIARY,
+        /**
+         * Specifies the property is previously resolved, e.g. in {@code resolve*Config} functions
```

Comment:
what's this asterisk for? `resolve*Config`

Reply:
It's just a placeholder for any generic resolve functions but it's just used in `resolveHttpAuthSchemeConfig()`  currently.